docs/cli/tag-management.md

```diff
@@ -197,3 +197,72 @@ spine1 :: VerifyUptime :: SUCCESS
 spine1 :: VerifyL3MTU :: SUCCESS
 spine1 :: VerifyUptime :: SUCCESS
 ```
+
+## Obtaining all configured tags
+
+As most ANTA commands accommodate tag filtering, this command is useful for enumerating all tags configured in the inventory. Running the `anta get tags` command will return a list of all tags configured in the inventory.
+
+### Command overview
+
+```bash
+Usage: anta get tags [OPTIONS]
+
+  Get list of configured tags in user inventory.
+
+Options:
+  -u, --username TEXT     Username to connect to EOS  [env var: ANTA_USERNAME;
+                          required]
+  -p, --password TEXT     Password to connect to EOS that must be provided. It
+                          can be prompted using '--prompt' option.  [env var:
+                          ANTA_PASSWORD]
+  --enable-password TEXT  Password to access EOS Privileged EXEC mode. It can
+                          be prompted using '--prompt' option. Requires '--
+                          enable' option.  [env var: ANTA_ENABLE_PASSWORD]
+  --enable                Some commands may require EOS Privileged EXEC mode.
+                          This option tries to access this mode before sending
+                          a command to the device.  [env var: ANTA_ENABLE]
+  -P, --prompt            Prompt for passwords if they are not provided.  [env
+                          var: ANTA_PROMPT]
+  --timeout FLOAT         Global API timeout. This value will be used for all
+                          devices.  [env var: ANTA_TIMEOUT; default: 30.0]
+  --insecure              Disable SSH Host Key validation.  [env var:
+                          ANTA_INSECURE]
+  --disable-cache         Disable cache globally.  [env var:
+                          ANTA_DISABLE_CACHE]
+  -i, --inventory FILE    Path to the inventory YAML file.  [env var:
+                          ANTA_INVENTORY; required]
+  --tags TEXT             List of tags using comma as separator:
+                          tag1,tag2,tag3.  [env var: ANTA_TAGS]
+  --help                  Show this message and exit.
+```
+
+### Example
+
+To get the list of all configured tags in the inventory, run the following command:
+
+```bash
+$ anta get tags
+Tags found:
+[
+  "BL",
+  "DC1",
+  "DC1-BL1",
+  "DC1-BL2",
+  "DC1-LEAF1A",
+  "DC1-LEAF1B",
+  "DC1-SPINE1",
+  "DC1-SPINE2",
+  "DC2",
+  "DC2-BL1",
+  "DC2-BL2",
+  "DC2-LEAF1A",
+  "DC2-LEAF1B",
+  "DC2-SPINE1",
+  "DC2-SPINE2",
+  "LEAF",
+  "SPINE"
+]
+```
+
+!!! note
+    ANTA automatically injects the device name as device tag.
```

Comment:
Not sure if it's necessary. It is mentionned above:
![image](https://github.com/user-attachments/assets/512ec38e-0643-4f9c-b379-49aace295f8c)